Web/starter_project_vanguard/data/home/activities.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
this should be assigned some name and exported to be used. i am not sure how you are using this